src/main/java/org/gridsuite/modification/server/entities/equipment/creation/CurrentTemporaryLimitCreationEmbeddable.java

```diff
@@ -25,6 +25,8 @@
 @Embeddable
 public class CurrentTemporaryLimitCreationEmbeddable {
 
+    // TODO : pas besoin d'id ici ??
```

Comment:
forgetten todo ?

Reply:
Indeed. It already worked without it. I was too suspicious because of a bug.